graylog2-web-interface/src/views/components/searchbar/AbsoluteTimeRangeSelector.jsx

```diff
@@ -36,7 +36,7 @@ const _isValidDateString = (dateString: string) => {
   }
   return DateTime.isValidDateString(dateString)
     ? undefined
-    : `Invalid date: ${dateString}`;
+    : `Format must be: ${DateTime.Formats.DATETIME}`;
```

Comment:
During testing, I noticed that we have a slight inconsistency. The message says `Format must be: YYYY-MM-DD HH:mm:ss`, but we do allow specifying milliseconds. So the message should be `Format must be: YYYY-MM-DD HH:mm:ss[.SSS]`. This would mean not being able to reuse the `DateTime.Formats.DATETIME` constant, but this is okay, as we should support a couple of formats and use a combined format for the error message.

A suggestion for a future improvement: I think we should be a bit more relaxed when parsing the date. Right now we expect to specify all units going down to seconds. Imo seconds should be optional, maybe it should even be possible to make hours & minutes optional as well.

Reply:
Good point, I've updated the error message.